ompi/mca/io/romio321/romio/adio/ad_gpfs/ad_gpfs_rdcoll.c

```diff
@@ -179,14 +179,14 @@ void ADIOI_GPFS_ReadStridedColl(ADIO_File fd, void *buf, int count,
     /* One-sided aggregation needs the amount of data per rank as well because the difference in
      * starting and ending offsets for 1 byte is 0 the same as 0 bytes so it cannot be distiguished.
      */
-    if ((gpfsmpio_read_aggmethod == 1) || (gpfsmpio_read_aggmethod == 2)) {
```

Comment:
This provides little conveniency while increasing the burden on the maintainer of the ROMIO integration. I would carefully assess the cost.